pandapipes/component_models/pump_component.py

```diff
@@ -102,11 +102,12 @@ def adaption_before_derivatives_hydraulic(cls, net, branch_pit, node_pit, idx_lo
         fluid = get_fluid(net)
         p_from = node_pit[from_nodes, PAMB] + node_pit[from_nodes, PINIT]
         # p_to = node_pit[to_nodes, PAMB] + node_pit[to_nodes, PINIT]
-        numerator = NORMAL_PRESSURE * pump_branch_pit[:, TINIT]
+        t_from = node_pit[from_nodes, TINIT_NODE]
+        numerator_from = NORMAL_PRESSURE * t_from
         v_mps = pump_branch_pit[:, VINIT]
         if fluid.is_gas:
             # consider volume flow at inlet
-            normfactor_from = numerator * fluid.get_property("compressibility", p_from) \
+            normfactor_from = numerator_from * fluid.get_property("compressibility", p_from) \
                               / (p_from * NORMAL_TEMPERATURE)
             v_mean = v_mps * normfactor_from
```

Comment:
I think we should go through the whole process again and check out how we get to the norm state (what temperature and pressure do we use?) and then how we can get back from there. Maybe v_mean also should be the mid between v_from and v_to.

Reply:
If we rename v_mean to v_from, it should be fine.